src/Native/Bootstrap/main.cpp

```diff
@@ -246,8 +246,11 @@ extern "C" void RhpUniversalTransition_DebugStepTailCall()
     throw "RhpUniversalTransition_DebugStepTailCall";
 }
 
+#if defined(_WASM_)
```

Comment:
This should not need ifdef. I think it can be `extern "C"` even without WASM.

Reply:
Done

Reply:
It turns out I had to also make the CppCodegen declaration ```extern "C"```, but that's probably a good thing anyway for easier linkage.